dev_deploy.sh

```diff
@@ -69,12 +58,8 @@ make build_dev
 # Change directory to the binary output directory
 cd bin
 
-# Start a Python HTTP server in the background to serve files
-python3 -m http.server "$PYTHON_PORT" &
-PYTHON_SERVER_PID=$!
-
-# Ensure that the Python server is terminated if the script exits unexpectedly
-trap "echo 'Terminating Python server...'; kill $PYTHON_SERVER_PID" EXIT
+# Copy the binary to the remote host
+cat jetkvm_app | ssh "${REMOTE_USER}@${REMOTE_HOST}" "cat > $REMOTE_PATH/jetkvm_app_debug"
```

Comment:
Why not use `scp`?

Reply:
I found out when writing https://medium.com/@brandontuttle/installing-tailscale-on-a-jetkvm-3c72355b7eb0 that dropbear doesn't support scp (notably sftp)

Reply:
I wonder if `-O` would work.

> Use  the  legacy  SCP protocol for file transfers instead of the SFTP protocol.`

Reply:
![image](https://github.com/user-attachments/assets/ad74d814-259d-4509-8628-ec6e3be4f926)
good thought, but I think that requires `scp` to installed on the remote